### ISSUE TITLE
Remove option "Lint only after change"

### ISSUE THREAD

**Issue by pchomik** [pchomik/linter-python#17]
Based on current linter and atom API it easy to discover two types of operation:
- lint after save (file is not modified)
- everything else
  "Lint after change and lint after save" are part of second group. The problem is that I'm not sure how to discover lint after change only.

Test scenario:

```
1. Open file
2. Add new char
3. Remove added char (linter will trigger lint process)
```

For such lint process we have lint after change but file status is "not modified". From plugin perspective this is very solid difference and because of that I'm thinking to remove option "Lint only after change" and keep two others.

**Comment by ghost** [comment]
Just understood the options and what the issue is. Perhaps it would be good to have flags that are a bit easier to understand? Something such as "Lint trigger": "File save", "File modify" (this would include saving and modifications).

**Comment by pchomik** [comment]
What do you think about "Lint trigger": "File saved", "File modified", "File saved or modified"? Is it clear enough for you what this option does/change?

**Comment by ghost** [comment]
I think that sounds nice and clear. On a slightly unrelated note, you did mention that different order of linters give slightly different results. Does the optimal order is already implemented in your package?

Also, thanks for the fast update of the package!

**Comment by ghost** [comment]
Also sorry to bother but did the last update solved the issue in the topic that was removed? I have just updated but get the same error message (opening the folder does solve it).

**Comment by pchomik** [comment]
Optimal order is used in package. I reported error to pylint because I discovered that some problems could be hidden. But this is related only to pylint.

**Comment by ghost** [comment]
Great, thanks for letting me know and making it easier for everyone to avoid the issues.

**Comment by pchomik** [comment]
Change available in 3.0.4
